OpenTrons/opentrons-api
# fix(protocol-designer): fix scroll on SelectPipettes screen

# Overview

Reverts overflow on SelectPipettes screen to `OVERFLOW_SCROLL`. Also, adds autoscroll to bottom to reveal tiprack selection if obscured.

## Test Plan and Hands on Testing

- create new protocol and land on pipette selection screen
- add pipette type and volume
- verify that if container for pipette fields selection sufficient for content height, you are autoscrolled to the bottom of the container upon selecting a volume

https://github.com/user-attachments/assets/e2d918b7-0ccf-446e-aaec-cc1fa1a080c4


## Changelog

- add autoscroll
- revert overflow value

## Review requests

see test plan

## Risk assessment

low
